site/docs/components/navigation-item/accessibility.mdx

```diff
@@ -10,56 +10,54 @@ data:
 
 ### Keyboard interactions
 
-#### Horizontal Navigation
+#### Horizontal navigation
```

Comment:
Is it worth having two sections here for vertical and horizontal navigation when the content is exactly the same except for Enter/Space where in the vertical orientation more children are revealed as the action and in the horizontal orientation it should display a list or mega menu?